plugins/inventory/cobbler.py

```diff
@@ -201,6 +210,10 @@ def parse(self, inventory, loader, path, cache=True):
         self.exclude_profiles = self.get_option('exclude_profiles')
         self.include_profiles = self.get_option('include_profiles')
         self.group_by = self.get_option('group_by')
+        if self.get_option('inventory_hostname') is not None:
+            self.inventory_hostname = self.get_option('inventory_hostname')
+        else:
+            self.inventory_hostname = 'hostname'
```

Comment:
I don't think there is a situation where this can be `None`.
```suggestion
        self.inventory_hostname = self.get_option('inventory_hostname')
```